sound/sequences/00_sound_player.s

```diff
@@ -2357,12 +2362,19 @@ chan_setinstr 7
 chan_setlayer 0, .layer_DDC
 chan_end
 
+// layer_portamento_voice and layer_note1_voice intentionally not used
+// NOTE: This particular sound has been modified for use with the DISABLE_MARIO_PITCH_SHIFTING define since it may otherwise sound bad left alone
 .layer_DDC:
-layer_transpose -2
+layer_transpose_voice -2
+#ifndef DISABLE_MARIO_PITCH_SHIFTING
 layer_somethingon
 layer_portamento 0x85, 38, 255
 layer_note1 41, 0xc, 115
 layer_note1 35, 0x12, 115
+#else
+layer_portamento 0x81, 41, 255
```

Comment:
why keep the portamento here? why is the pitch 38 instead of 39? and the voice macro is probably still fine to use for consistency

Reply:
I made it like this because using the voice macro here (and in the other two cough spots) makes all three of them exactly the same. Idk, it just sounds odd without the variation.

Reply:
well its not really supposed to sound good its just supposed to be easy to drop new sfx in which likely would already account for making them sound good so idk about this one, although idk how the cough sfx works

Reply:
Well the first cough sound is pretty much unmodified anyway, so one of them will already match up with the original sample. Same goes for panting.